CHANGES.rst

```diff
@@ -23,6 +23,9 @@ Other changes
 -------------
 
 * The FAQ page was moved from the GitHub Wiki to the regular documentation.
+* ``np.long_t`` and ``np.ulong_t`` were removed, synching Cython with upstream
```

Comment:
```suggestion

* ``np.long_t`` and ``np.ulong_t`` were removed, synching Cython with upstream
```

(Don't bother accepting this change now and causing the tests to be re-run. I'll do it just before merging)